src/main/java/com/example/domainmodelopdrachtspringbootles11/model/Television.java

```diff
@@ -0,0 +1,176 @@
+package com.example.domainmodelopdrachtspringbootles11.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name="televisions")
+public class Television {
```

Comment:
Je hebt deze entity nog niet gebruikt in je controller.

Reply:
Nu wel :)